packages/fxa-graphql-api/src/lib/server.ts

```diff
@@ -31,20 +32,24 @@ export async function createServer(
   logger: Logger,
   context?: () => object
 ): Promise<ApolloServer> {
-  setupDatabase(config.database);
+  const fetchUserFn = Container.get(userLookupFnContainerKey) as typeof fetchUserByToken;
   const schema = await TypeGraphQL.buildSchema({
     container: Container,
     resolvers: [AccountResolver],
   });
   const debugMode = config.env !== 'production';
-  const defaultContext = ({ req }: any) => {
-    const authUser = req.headers[config.authHeader.toLowerCase()];
+  const defaultContext = async ({ req }: any) => {
+    const bearerToken = req.headers[config.authHeader.toLowerCase()];
+    const authUser = await fetchUserFn(bearerToken);
+    if (!authUser) throw new AuthenticationError('You must be logged in');
+
     return {
       authUser,
       logAction: (action: string, options?: object): void => {
-        logger.info(action, { authUser, ...options });
+        logger.info(action, { uid: authUser.userId, ...options });
       },
       logger,
+      token: bearerToken,
```

Comment:
Copied over because we'll need to pass it onto the auth-server to access its API.

Reply:
Feels fine for now, Apollo has [some tooling ](https://www.apollographql.com/docs/apollo-server/data/data-sources/#rest-data-source)to make it easier to link portions of a graph object to remote API calls, so we'll likely need to plumb that through there later.

Reply:
Hmm the doc's examples are using `this.context.token`, which is this I believe.